akka-distributed-cluster-docs/src/main/paradox/sample.md

```diff
@@ -0,0 +1,48 @@
+# Sample - Distributed Shopping Cart
+
+For a more hands on sample we have implemented to variations on a distributed shopping cart, where a user can add and
+remove items and finally check the selected goods out. This is a somewhat synthetic example, but, it is a domain we expect 
+you will quickly understand, so that we can focus on explaining the various features of Akka Distributed Cluster. 
+
+## Active-active
+
+For an Active-Active shopping cart, the same service will run in different data centers or cloud regions.
+
+### Filtering
+
+The replicated shoppig cart is set up to only replicate carts containing 10 or more items.
+
+It also contains a consumer side filter, to explicitly exclude specific shopping cart ids requested by a client.
```

Comment:
The RES sample only has producer filter based on tags.

What you were thinking about was the original consumer filter for grpc Projection sample.
I changed that because I thought it wasn't good and opened up for problems like when to remove such entity id filters to avoid growing that list infinitely. The consumer filter is also based on tags. Pretty much the same thing as the producer filter.

Reply:
Hmm, when I look at the sample, I see `excludeCart` in `ShoppingCartServiceImpl`?

Reply:
um, must be some leftover from my testing. Please remove that.

Reply:
b6e4440c